docs/config/_default/config.toml

```diff
@@ -16,7 +16,14 @@ defaultContentLanguageInSubdir = false
 # Useful when translating.
```

Comment:
Here with this file I've moved to a structure that allows hugo to override settings based on the "environment" selected when doing the building.

_default represents our "production" world -- the base set.